src/pages/index.module.scss

```diff
@@ -0,0 +1,96 @@
+.heroElectron {
+    color: var(--electron-color-light);
+    background: var(--electron-color-dark);
+}
+
+@keyframes hero-animation {
```

Comment:
If we have to use camel casing for the class names should we use it also everywhere in the CSS?

Reply:
That's a good point. I'll amend this to keep it uniform :)

Reply:
fb85d488820c3e3bfbe0cc3680237abd8e961543